contentgrid-spring-swagger/src/main/java/com/contentgrid/spring/swaggerui/SwaggerUIInitializerController.java

```diff
@@ -0,0 +1,31 @@
+package com.contentgrid.spring.swaggerui;
```

Comment:
Isn't the resource file served automatically when it's put on the classpath in the correct location?

Reply:
It is, but in this case it is problematic:
- either you put it in a versioned folder, like /webjars/swagger-ui/4.11.1/ and you have problems when the dependency gets updated
- if you put it one level higher, the webjar version detection breaks

Reply:
That is something useful to put in a comment on the class then?

Reply:
See updated javadoc